src/content/docs/wallets.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Web Monetization-compatible wallets
+lastUpdated: 2025-02-18
+---
+
+import { Steps, Tabs, TabItem } from '@astrojs/starlight/components'
+import { LinkOut } from '@interledger/docs-design-system'
+
+The following providers offer Web Monetization-compatible digital wallets. You must have a wallet address or payment pointer from a compatible wallet to send and receive payments.
+
+Be sure to select a wallet that's available in your **region** and supports your preferred **currency**. See the [Limitations](#limitations) section for additional details.
+
+## Wallet providers
+
+| Provider | Payment address label | Available in | Supported fiat currencies |
+| -------- | --------------------- | ------------ | -------------------- |
+| <LinkOut href="https://interledger.app/">Interledger Wallet</LinkOut> | Wallet address | USA, EU, Canada, South Africa | USD, EUR, CAD, ZAR |
+| <LinkOut href="https://gatehub.net">GateHub</LinkOut> | Payment pointer | EU | EUR |
+
+## Interledger test wallet
+
+The Interledger test wallet provides a great way to test Web Monetization without using real money. The test wallet is an open-source application developed and maintained by the [Interledger Foundation](https://interledger.org).
+
+Check out [this page](/tutorials/test-wallet-sign-up) for the steps to create an account on the Interledger test network.
```

Comment:
```suggestion
[Create an account on the Interledger test network](/tutorials/test-wallet-sign-up) to test Web Monetization without real money.
```

Ref:
- https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/
- https://www.a11yproject.com/posts/creating-valid-and-accessible-links/#write-helpful-link-text
- https://heyoka.medium.com/dont-use-click-here-f32f445d1021

Reply:
Yeah I know about that, I just missed it lol Thank you